Update WPT webrtc-extensions up to d643cf9

youennf opened the pull request:
#### 104c8d43deff196f2344f95fc028ecc0f92b5e1a
<pre>
Update WPT webrtc-extensions up to d643cf9
<a href="https://rdar.apple.com/123240144">rdar://123240144</a>
<a href="https://bugs.webkit.org/show_bug.cgi?id=269715">https://bugs.webkit.org/show_bug.cgi?id=269715</a>

Reviewed by Eric Carlson.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpParameters-codec-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpParameters-codec.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpParameters-maxFramerate-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpParameters-maxFramerate.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-jitterBufferTarget-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-jitterBufferTarget-stats-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-jitterBufferTarget-stats.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-jitterBufferTarget.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-playoutDelayHint-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-playoutDelayHint.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpSynchronizationSource-captureTimestamp.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpSynchronizationSource-senderCaptureTimeOffset.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpTransceiver-headerExtensionControl-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpTransceiver-headerExtensionControl.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/transfer-datachannel-service-worker.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/transfer-datachannel-service-worker.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/transfer-datachannel-service-worker.js: Added.
(onmessage):
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/275033@main">https://commits.webkit.org/275033@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d431c0e874f93ecd3903445d792761f77532023b

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/40712 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/19725 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/43090 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/43268 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/36802 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/22701 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/17056 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/33766 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/41286 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/16662 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/35088 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/14363 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/14446 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/36077 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/44542 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/36922 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/36389 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/40130 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/15527 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/12734 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/38471 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/17146 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/35359 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/9119 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/17197 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/16790 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

eric-carlson reviewed (approved):
rs=me once the bots are happy

youennf committed:
Update WPT webrtc-extensions up to d643cf9
rdar://123240144
https://bugs.webkit.org/show_bug.cgi?id=269715

Reviewed by Eric Carlson.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpParameters-codec-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpParameters-codec.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpParameters-maxFramerate-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpParameters-maxFramerate.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-jitterBufferTarget-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-jitterBufferTarget-stats-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-jitterBufferTarget-stats.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-jitterBufferTarget.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-playoutDelayHint-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpReceiver-playoutDelayHint.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpSynchronizationSource-captureTimestamp.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpSynchronizationSource-senderCaptureTimeOffset.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpTransceiver-headerExtensionControl-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/RTCRtpTransceiver-headerExtensionControl.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/transfer-datachannel-service-worker.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/transfer-datachannel-service-worker.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/transfer-datachannel-service-worker.js: Added.
(onmessage):
* LayoutTests/imported/w3c/web-platform-tests/webrtc-extensions/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/275033@main
